src/components/api_reference_page.jsx

```diff
@@ -15,6 +15,7 @@ export default class APIReferencePage extends React.Component {
   }
 
   render() {
+    let defaultName = "achievements";
```

Comment:
This is needed for some bizarre reason. I have no idea why the logic is what is but :P